beaker-vue/src/components/cell/BeakerCodeCell.vue

```diff
@@ -248,7 +282,7 @@ export default {
     }
     .cm-focused {
         outline: none;
-        border: 1px solid var(--purple-200);
```

Comment:
Drive-by: this rule wasn't doing anything after the vue upgrade. From this focusing on a code cell now adds the subtle purple border instead of making the gray border disappear when selected.